e2e-polybft/framework/test-cluster.go

```diff
@@ -429,6 +441,15 @@ func NewTestCluster(t *testing.T, validatorsCount int, opts ...ClusterOption) *T
 			args = append(args, "--mintable-native-token")
 		}
 
+		if len(cluster.Config.BurnContracts) != 0 {
+			for block, addr := range cluster.Config.BurnContracts {
+				args = append(args, "--burn-contract", fmt.Sprintf("%d:%s", block, addr))
+			}
+		} else {
+			// London hardfork is enabled by default so there must be a default burn contract
+			args = append(args, "--burn-contract", "0:0x0000000000000000000000000000000000000000")
```

Comment:
Default value?

Reply:
Not sure, I would explicitly specify the contract address in order to avoid confusions